### PR TITLE
feat/CIV-237-II

### PR DESCRIPTION
**Pull request by indu2019**
LR vs LR - Admits Part: Screen to capture the defendant's to capture if defendant is paying money as a result of court orders.

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-237

### Change description ###
LR vs LR - Admits Part: Screen to capture the defendant's to capture if defendant is paying money as a result of court orders.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: ccd-definition/ComplexTypes/PayingExtraInfo.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "ID": "PayingExtraInfo",
+    "ListElementCode": "claimNumberText",
+    "FieldType": "Text",
+    "FieldTypeParameter": "",
```
Comment (dtortolaV1) [inline ccd-definition/ComplexTypes/PayingExtraInfo.json:6]: Is this line needed?, with FieldType text, fieldTypeParameter should be ignored anyway

File: ccd-definition/ComplexTypes/PayingExtraInfo.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "ElementLabel": "Claim number",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "PayingExtraInfo",
+    "ListElementCode": "amountOwed",
+    "FieldType": "MoneyGBP",
+    "FieldTypeParameter": " ",
```
Comment (dtortolaV1) [inline ccd-definition/ComplexTypes/PayingExtraInfo.json:14]: fieldTypeParameter may be unnecessary

File: ccd-definition/ComplexTypes/PayingExtraInfo.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "ElementLabel": "Amount owed",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "PayingExtraInfo",
+    "ListElementCode": "monthlyInstalmentAmount",
+    "FieldType": "MoneyGBP",
+    "FieldTypeParameter": " ",
```
Comment (dtortolaV1) [inline ccd-definition/ComplexTypes/PayingExtraInfo.json:22]: fieldTypeParameter may be unnecessary

File: ccd-definition/ComplexTypes/PayingMoneyAsAResultOfCourtOrder.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "ListElementCode": "payingMoneyDetailsLabel",
+    "FieldType": "Label",
+    "ElementLabel": "## Is ${respondent1.partyName} paying money as a result of any court order? \n",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "PayingMoneyAsAResultOfCourtOrder",
+    "ListElementCode": "payingDetailsRequired",
+    "FieldType": "YesOrNo",
+    "ElementLabel": " ",
```
Comment (dtortolaV1) [inline ccd-definition/ComplexTypes/PayingMoneyAsAResultOfCourtOrder.json:13]: If you set the previous Label's ElementLabel here as this ElementLabel, does the result look the same? If so, you can save the payingMoneyDetailsLabel element

**Comment by indu2019** [comment]
closing this PR

**Pull request closed without merging**